install/migrations/update_9.5.6_to_9.5.7.php

```diff
@@ -84,6 +84,37 @@ function update956to957()
     );
     /** /Replace -1 values for glpi_networkportaliases.networkports_id_alias field */
 
+    /** Fix duplicate date in glpi_items_operatingsystems table */
+    // First, find item that contains duplicated data
+    $data = $DB->request([
+        'SELECT' => [
+            'MAX' => 'id as valid_entry_id',
+            'items_id',
+            'itemtype',
+        ],
+        'FROM' => 'glpi_items_operatingsystems',
+        'WHERE' => [
+            'operatingsystems_id' => [-1, 0]
+        ],
+        'GROUPBY' => [
+            'items_id',
+            'itemtype'
```

Comment:
But there should be one OS per computer only, with your proposal, you can have several, that does not seems correct

Reply:
This is probably another problem. Here we just try to fix the migration that fails on unicity.

If we should have only one OS per Asset, we should probably change the unicity key to only include `itemtye, items_id`, but it may require to handle this globally (we have have to fix some plugins for instance).

Reply:
OK, anayway that solve the initial issue on migration